doc/nrf/ext_comps.rst

```diff
@@ -0,0 +1,15 @@
+.. _ext_components:
+
+Using external components
+#########################
+
+The following user guides show how you can use external components in your application:
```

Comment:
```suggestion
The following user guides show how you can use external components in your application.
```
There's the Subpages header after the colon, so I'd suggest a period here. We don't introduce headers with colons.